app.js

```diff
@@ -1 +1,35 @@
-console.log("Todo List NodeJS!")
\ No newline at end of file
+
+
+const readline = require('readline'); //console.info(readline);
+let userQuestions = readline.createInterface(process.stdin,process.stdout);
+let decided = readline.createInterface(process.stdin, process.stdout);
+
+function getUserWant(){
+    "use strict";
+    const activities = [{
+        monday : "playing soccer",
+        tuesday : "playing volley",
+        wednesday : "playing basketball",
+        thursday : "playing shooter game",
+        friday : "walk around with pets",
+        saturday : "hanging out with my Boo or with homies",
+        sunday : "sleeping over all times"
+    }];
+
+    //TODO :  User just only typing the keys with key named day that has been created on above variable object*/
+    // TODO : Control user that want continue or stop
```

Comment:
Why do you need to set a goal to make a feature that we don't need? Please delete this.

Reply:
This include the TODO on line 19